lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -380,7 +380,7 @@
   vapp_properties:
     description:
     - A list of vApp properties
-    - 'For full list of attributes and types refer to: U(https://github.com/vmware/pyvmomi/blob/master/docs/vim/vApp/PropertyInfo.rst)'
+    - 'For full list of attributes and types refer to: U(https://vdc-download.vmware.com/vmwb-repository/dcr-public/6b586ed2-655c-49d9-9029-bc416323cb22/fa0b429a-a695-4c11-b7d2-2cbc284049dc/doc/index.html)'
```

Comment:
@sabre1041 Thanks for the PR. Could you please point the exact location of vAPP properties list page so that users will directly go that link? Right now, it is pointing to index.html.

Also, add a changelog entry [here](https://github.com/ansible/ansible/tree/devel/changelogs/fragments) so that we can backport this change to older release.

Thanks,

Reply:
@Akasurde done